Common/Gfx/Shape.h

```diff
@@ -1,20 +1,30 @@
-#pragma once
+/* Copyright (C) 2013 Rainmeter Project Developers
+*
+* This Source Code Form is subject to the terms of the GNU General Public
+* License; either version 2 of the License, or (at your option) any later
+* version. If a copy of the GPL was not distributed with this file, You can
+* obtain one at <https://www.gnu.org/licenses/gpl-2.0.html>. */
```

Comment:
There should be a space before the `*`. VS removes annoyingly removes them... Could you also change the year to 2016?